mmv1/third_party/terraform/tpgresource/import.go

```diff
@@ -25,10 +25,7 @@ func ParseImportId(idRegexes []string, d TerraformResourceData, config *transpor
 			log.Printf("[DEBUG] Could not compile %s.", idFormat)
 			return fmt.Errorf("Import is not supported. Invalid regex formats.")
 		}
-		identity, err := d.Identity()
-		if err != nil {
-			return err
-		}
+		identity, _ := d.Identity()
```

Comment:
Should we have a nil check here?

Reply:
yeah we should include at least a check so that it outputs the err in the debug log